modular_citadel/code/modules/client/loadout/donor.dm

```diff
@@ -13,3 +13,18 @@
 /obj/item/storage/box/large/custom_kit/mrmannstealthsuit/PopulateContents()
 	new /obj/item/clothing/under/f13/chinasuitcosmetic(src)
 	new /obj/item/clothing/head/f13/chinahelmetcosmetic(src)
+
+/datum/gear/donator/kits/midgetdragonvest
+	name = "Great Beat Vest Bundle"
+	path = /obj/item/clothing/under/f13/bearvest
+	category = LOADOUT_CATEGORY_UNIFORM
+	subcategory = LOADOUT_SUBCATEGORY_UNIFORM_GENERAL
+	ckeywhitelist = list("midgetdragon")
+
+/obj/item/storage/box/large/custom_kit/midgetdragonvest
+	name = "Great Bear Vest Bundle"
+	desc = "A vest, scarf and a pair of ripped jeans."
+	w_class = WEIGHT_CLASS_BULKY
+
+/obj/item/storage/box/large/custom_kit/mrmannstealthsuit/PopulateContents()
+	new /obj/item/clothing/under/f13/bearvest(src)
```

Comment:
Does not match the item description, missing a scarf and a pair of ripped jeans.

If it's only one donor item then adding a kit for it is redundant since you can just use that item path in the gear datum. As you're already doing.

Reply:
i am not a coder and will not be on PC to make changes for most of today, neither of these sound like an actual issue, can this just be merged? It is for a donator and they have been waiting for days already